NOTICE-binary

```diff
@@ -4732,11 +4851,6 @@ Copyright 2014-2019 The Apache Software Foundation
 flink-fs-hadoop-shaded
 Copyright 2014-2019 The Apache Software Foundation
 
-This project includes software developed at
-The Apache Software Foundation (http://www.apache.org/).
-
-This project bundles the following dependencies under the Apache Software License 2.0 (http://www.apache.org/licenses/LICENSE-2.0.txt)
-
```

Comment:
Why did you remove these lines?

Reply:
I didn't do any manual changes; I just built everything, ran the collect_licenses script and removed some unrelated changes.

I'll double check whether an older jar sneaked in, and how the notice file in the oss jar looks like. AFAIK the shade-plugin isn't deduplicating entries, so this is indeed odd.

Reply:
nope, happens on a clean build. Is also consistent with the entries for other filesystems that bundle flink-fs-shaded-hadoop.

Reply:
Ok, then I guess it should be correct.